features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/IncomingCallActivity.kt

```diff
@@ -124,93 +133,137 @@ class IncomingCallActivity : AppCompatActivity() {
     private fun onCancel() {
         activeCallManager.hungUpCall()
     }
+}
 
-    @Composable
-    private fun IncomingCallScreen(
-        notificationData: CallNotificationData,
-        onAnswer: (CallNotificationData) -> Unit,
-        onCancel: () -> Unit,
-    ) {
-        ElementTheme {
+@Composable
+private fun IncomingCallScreen(
```

Comment:
Maybe extract to its own file?

Reply:
I didn't thought it was worth it as it's not likely that we'll use it anywhere else, but sure I can do it.

Reply:
Yes, thanks, I like to have the files containing an Activity as short as possible.